prbt_hardware_support/include/prbt_hardware_support/param_names.h

```diff
@@ -27,6 +27,7 @@ static const std::string PARAM_MODBUS_SERVER_IP_STR {"modbus_server_ip"};
 static const std::string PARAM_MODBUS_SERVER_PORT_STR {"modbus_server_port"};
 static const std::string PARAM_MODBUS_RESPONSE_TIMEOUT_STR {"modbus_response_timeout"};
 static const std::string PARAM_MODBUS_TOPIC_NAME_STR {"modbus_topic_name"};
```

Comment:
rename this to PARAM_MODBUS_READ_TOPIC_NAME_STR
Not sure about  the topic name itself. maybe it should stay like this, for compatibility. Maybe do a search and replace in the repo to do modbus_topic_name -> modbus_read_topic_name"